### PR TITLE
AddProjectMemberDialog: add bottom border

### LINKED ISSUE #7101: Add Project Member permissions card gradient border is clipped
**Setup**
- Rancher version: 2.6.9

**Describe the bug**
Bottom border of permissions card is clipped

image

image

**To Reproduce**
- Edit a project
- Add Membmer

### PR DESCRIPTION
**Pull request by torchiaf**
Signed-off-by: Francesco Torchia <francesco.torchia@suse.com>

### Summary
Fixes rancher/dashboard#7101

### CODE REVIEW

File: shell/dialog/AddProjectMemberDialog.vue [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default {
   <Card class="prompt-rotate" :show-highlight-border="false" :sticky="true">
     <h4 slot="title" class="text-default-text" v-html="t('addProjectMemberDialog.title')" />
 
-    <div slot="body" class="pl-10 pr-10">
+    <div slot="body" class="pl-10 pr-10 pb-10">
```
Comment (cnotv) [inline shell/dialog/AddProjectMemberDialog.vue:82]: I think @nwmac [mentioned here](https://github.com/rancher/dashboard/issues/7101#issuecomment-1286255377) that we should replace the box shadow with the border.

**Review by cnotv — CHANGES_REQUESTED** [review]
Please check the requirement or eventually ask for clarification if want to double check it.

File: shell/dialog/AddProjectMemberDialog.vue (hunk 1 above)
Comment (torchiaf) [reply shell/dialog/AddProjectMemberDialog.vue:82]: I didn't notice the requirement about border, thanks!

**Review by cnotv — APPROVED** [review]
LGTM

**Pull request merged**